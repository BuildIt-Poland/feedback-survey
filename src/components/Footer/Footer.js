import React from 'react';
import styled from 'styled-components';
import { Grommet, Box, Text } from 'grommet';

import { colorDarkBlue, colorWhite, fontSmall } from '../../styles/designTokens';

const theme = {
  text: {
    medium: {
      size: fontSmall
    }
  }
};

const Wrapper = styled.footer`
  grid-area: footer;
  background: ${colorDarkBlue};
  color: ${colorWhite};
  text-align: center;
  padding: 1rem 2rem;
`;

const Footer = () => (
  <Wrapper>
    <Text size="medium">
      Randstad Sourceright (Hong Kong SAR) EA Licence No. 57155 | Randstad Sourceright (Singapore) EA
    </Text>
  </Wrapper>
);

export default Footer;
