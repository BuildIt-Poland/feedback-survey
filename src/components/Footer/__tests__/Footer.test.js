import React from 'react';
import { create } from 'react-test-renderer';

import Footer from '../Footer';

describe('COMPONENT - Footer', () => {
  it('renders correctly', () => {
    const component = create(<Footer />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
