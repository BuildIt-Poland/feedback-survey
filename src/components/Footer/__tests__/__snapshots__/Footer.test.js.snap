// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Footer renders correctly 1`] = `
<footer
  className="sc-bwzfXH hACxDC"
>
  <p
    className="sc-bdVaJa gxYkJN"
  >
    Randstad Sourceright (Hong Kong SAR) EA Licence No. 57155 | Randstad Sourceright (Singapore) EA
  </p>
</footer>
`;
